deepspeech/main.go

```diff
@@ -70,18 +61,6 @@ func main() {
 		return
 	}
 
-	if err := m.SetBeamWidth(beamWidth); err != nil {
-		log.Fatal("Failed setting beam width: ", err)
-	}
-	if *scorer != "" {
-		if err := m.EnableExternalScorer(*scorer); err != nil {
```

Comment:
i think that removing this call to `EnableExternalScorer` was accidental, right?

(and just to check, are `SetBeamWidth` and `SetScorerAlphaBeta` now unnecessary in most cases, e.g. because reasonable defaults are used?)

Reply:
No, scorer now bundles those values.

Reply:
Beam width and alpha/bêta are bundled yes.

Reply:
Scorer should be put back.

Reply:
Done https://github.com/asticode/go-astideepspeech/commit/1e4e7bdece5230c6ba16275f901a51a4c53fc5ec